app/Http/Controllers/SettingsController.php

```diff
@@ -192,7 +192,7 @@ public function postSaveFirstAdmin(SetupUserRequest $request) : RedirectResponse
         $settings->next_auto_tag_base = 1;
         $settings->auto_increment_assets = $request->input('auto_increment_assets', 0);
         $settings->auto_increment_prefix = $request->input('auto_increment_prefix');
-        $settings->zerofill_count = $request->input('zerofill_count', 5);
+        $settings->zerofill_count = $request->input('zerofill_count') ?: 0;
```

Comment:
I think just:

`$settings->zerofill_count = $request->input('zerofill_count', 0);`

would work, no?

Reply:
I thought so too but when I tried it the result of the function was `null` which triggered an exception.

Reply:
Ah I figured it out. 

These are checkboxes on the front end and if they are unchecked the browser does not include them in the payload:
```
$settings->full_multiple_companies_support = $request->input('full_multiple_companies_support', 0);
$settings->auto_increment_assets = $request->input('auto_increment_assets', 0);
```

This is a text input so the browser always includes it but Laravel sets it to null if nothing was input:
```
$settings->zerofill_count = $request->input('zerofill_count', 0);
```
So it ends up using `null` because it _is_ in the request.